packages/console/src/onboarding/pages/SignInExperience/components/InspireMe/index.module.scss

```diff
@@ -24,8 +24,17 @@
   }
 
   .button {
-    &:not(:disabled):not(:active):hover {
-      background: var(--color-layer-1);
+    &:not(:disabled) {
+      &:not(:active),
+      &:active {
+        &:hover {
+          background: var(--color-layer-1) center / 90% no-repeat url('../../../../assets/images/fireworks.png');
+        }
+      }
```

Comment:
I don't think the `&:active` is needed here. So probably just
```css
&:not(:active):hover {
  background: blah blah
}
```

Reply:
Since the `active` is used in the `Button` component, we need to use it to overwrite the style.